package.json

```diff
@@ -2,15 +2,7 @@
   "name": "visible-wordpress",
   "version": "1.0.0",
   "scripts": {
-    "export-db": "docker exec $(docker ps -lq) /bin/bash -c \"sudo -u www-data wp db export /data/database.sql\"",
-    "import-db": "docker exec $(docker ps -lq) /bin/bash -c \"sudo -u www-data wp db import /data/database.sql\" && npm run search-replace",
-    "update-plugins": "docker exec $(docker ps -lq) /bin/bash -c \"sudo -u www-data wp plugin update --all\"",
-    "update-themes": "docker exec $(docker ps -lq) /bin/bash -c \"sudo -u www-data wp theme update --all\"",
-    "plugin-install": "docker exec $(docker ps -lq) /bin/bash -c \"sudo -u www-data wp plugin install $PLUGIN\"",
-    "plugin-update": "docker exec $(docker ps -lq) /bin/bash -c \"sudo -u www-data wp plugin update $PLUGIN\"",
-    "theme-update": "docker exec $(docker ps -lq) /bin/bash -c \"sudo -u www-data wp theme update $THEME\"",
-    "theme-install": "docker exec $(docker ps -lq) /bin/bash -c \"sudo -u www-data wp theme install $THEME\"",
-    "search-replace": "docker exec $(docker ps -lq) /bin/bash -c \"sudo -u www-data wp search-replace 'http://www.oldurl.com' 'http://localhost:8080' --skip-columns=guid\""
+    "wp": "wpcmd() { docker exec $(docker ps -lq) /bin/bash -c \"sudo -u www-data wp $(echo $@)\"; };wpcmd "
```

Comment:
@karellm Bingo. This is the winner... Using this new run script, wp-cli feels like it's running natively but runs within the wordpress container.

So, for example...

- `npm run wp plugin install <some-plugin>` installs `<some-plugin>` inside the container
- `npm run wp db import /data/database.sql` imports a the database
...etc, etc, etc.


If you're cool with this and the last change, let me know and I'll rebase. :+1: